docs/pages/eas/workflows/syntax.mdx

```diff
@@ -696,20 +721,51 @@ jobs:
       # @end #
 ```
 
-### `jobs.<job_id>.steps`
+### `jobs.<job_id>.image`
 
-A job contains a sequence of tasks called `steps`. Steps can run commands.
+Specifies the VM image to use for the job. See [Infrastructure](/build-reference/infrastructure/) for available images.
 
 ```yaml
 jobs:
   my_job:
     # @info #
-    steps:
-      - name: My first step
-        run: echo "Hello World"
+    image: auto | string # optional, defaults to 'auto'
+    # @end #
+```
+
+### `jobs.<job_id>.runs_on`
+
+Specifies the worker that will execute the job. Available only on custom jobs.
+
+```yaml
+jobs:
+  my_job:
+    # @info #
+    runs_on: linux-medium | linux-large |
+      linux-medium-nested-virtualization |
+      linux-large-nested-virtualization |
+      macos-medium | macos-large # optional, defaults to linux-medium
     # @end #
 ```
 
+{/* vale off */}
+
+| Worker                             | vCPU | Memory (GiB RAM) | SSD (GiB) | Notes                             |
+| ---------------------------------- | ---- | ---------------- | --------- | --------------------------------- |
+| linux-medium                       | 4    | 16               | 14        | Default worker.                   |
+| linux-large                        | 8    | 32               | 28        |                                   |
+| linux-medium-nested-virtualization | 4    | 16               | 14        | Allows running Android Emulators. |
+| linux-large-nested-virtualization  | 4    | 16               | 28        | Allows running Android Emulators. |
```

Comment:
I think the SSD values are wrong. Unless I'm missing something, we're giving Linux workers 100 GB of disk space? cc @randomhajile 

https://github.com/expo/workflow-orchestration/blob/4339ef73525edde7cb252f311fd22565425de844/k8s/production/drone.toml#L8